docs/reference/client-config.md

```diff
@@ -21,38 +20,25 @@ but the path to a different configuration file can be specified using the
 
 ## Overview of the file
 
-A configuration file consists of the following sections:
+In addition to the configuration file format, please see the optional password
+[environment variables](#environment-variables-optional) that the Notary client
+can take for ease of use.
 
-<table>
-	<tr>
-		<td><a href="#trust-dir-section-optional">trust_dir</a></td>
-		<td>TUF key and metadata directory</td>
-		<td>(optional)</td>
-	</tr>
-	<tr>
-		<td><a href="#remote-server-section-optional">remote_server</a></td>
-		<td>remote Notary Server configuration</td>
-		<td>(optional)</td>
-	</tr>
-</table>
+Here is a full client configuration file example; please click on the top level
+JSON keys to learn more about the configuration section corresponding to that key:
 
-In addition please see the optional password [environment
-variables](#environment-variables-optional) that the Notary client can take for
-ease of use.
 
-An example (full) clietn configuration file.
-
-```json
+<pre><code class="language-json">
```

Comment:
yup, it looks pretty good when I generate the docs :)

![screen shot 2016-03-01 at 3 10 31 pm](https://cloud.githubusercontent.com/assets/1865981/13445217/d147b906-dfbf-11e5-8ac5-8b82734b0187.png)